docs/md/dev-process.md

```diff
@@ -283,3 +283,59 @@ It takes a vector of `var`s like this.
 ``` clojure
 (comment (test-vars [#'test-something]))
 ```
+
+### No tests found
```

Comment:
Thank you for adding this!! I ran into this without a clue just yesterday!!

Reply:
A first time for me as well!  It lulls you into a false sense of security.